chess/frontend/gui/src/modules/Game/subcomponents/Board.jsx

```diff
@@ -57,23 +54,23 @@ export class Board extends React.Component {
     }
     return (
       <div className="tri-list">
-        <Clock
+        {/* <Clock
           time={this.props.blackTime}
           turn={this.props.turn}
           player={BLACK}
           move={this.props.move}
-        />
+        /> */}
```

Comment:
Is this commented out? If so, let's remove it

Reply:
I don't want to remove it yet, just in case if I can fix the clock issues later